### ISSUE TITLE
toast-cmb 2.3.14 not compatible with the current version of litebird_sim

### ISSUE THREAD

**Issue by sgiardie** [litebird/litebird_sim#140]
the installed version of toast misses some files and leads to errors when importing litebird_sim

**Comment by ziotom78** [comment]
Hi @sgiardie , I believe that you can fix the problem by simply running this program:

    pip install --upgrade -r requirements.txt

Please tell us if it works.

**Comment by tskisner** [comment]
Running a test from a fresh virtualenv the file is there:
```
$>  virtualenv toast_test

$>  source toast_test/bin/activate

$>  pip install toast

$>  ls toast_test/lib/python3.9/site-packages/toast/todmap/mapmaker.py 
  toast_test/lib/python3.9/site-packages/toast/todmap/mapmaker.py
```
And of course when building the toast wheels and before uploading to PyPI we install the wheel and run the unit test suite.  So packaging errors should be caught (although there is always the possibility of bugs in the process...)

**Comment by sgiardie** [comment]
Hi all, I solved very easily by doing `pip uninstall toast` (which was actually already at the 2.3.14 version) and then `pip install toast`. It has installed the same version, but this time with all the files needed, so I don't have anymore problems when importing litebird_sim in python. On the cluster, I tried creating a new conda environment from scratch with python=3.8.10, then doing `pip install .` in the litebird_sim directory. It installed the right version of toast without any file missing. I don't know exactly what was wrong in my local environment.
@paganol was experiencing the same error though.
